modules/rostests/apitests/apphelp/apphelp.c

```diff
@@ -869,6 +870,33 @@ static void test_ApplicationAttributes(void)
     DeleteFileA("testxx.exe");
 }
 
+/* Showing that SdbGetAppPatchDir returns HRESULT */
+static void test_SdbGetAppPatchDir(void)
+{
+    WCHAR Buffer[MAX_PATH];
+    HRESULT hr;
+    int n;
+
+    hr = pSdbGetAppPatchDir(NULL, NULL, 0);
+    ok_hex(hr, S_FALSE);
```

Comment:
Hmm. Are you able to make it return any other code? S_OK/S_FALSE isn't exactly the most convincing case for HRESULT ;)

Reply:
shit this is an old commit, i must have pushed the wrong one...